# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mangohud/sde-version.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- MangoHud-v0.7.0/src/overlay.cpp.orig	2024-01-07 14:18:45.647458553 +0100
+++ MangoHud-v0.7.0/src/overlay.cpp	2024-01-07 14:19:10.032720126 +0100
@@ -940,7 +940,7 @@
       trim(cpu);
       kernel = exec("uname -r");
       trim(kernel);
-      os = exec("sed -n 's/PRETTY_NAME=\\(.*\\)/\\1/p' /etc/*-release");
+      os = exec("cat /etc/SDE-VERSION");
       os.erase(remove(os.begin(), os.end(), '\"' ), os.end());
       trim(os);
       cpusched = read_line("/sys/devices/system/cpu/cpu0/cpufreq/scaling_governor");
